pgbot/commands/admin/emsudo.py

```diff
@@ -858,19 +858,23 @@ async def cmd_emsudo_replace(
     @add_group("emsudo", "edit")
     async def cmd_emsudo_edit(
         self,
-        msg: discord.Message,
+        msg: Union[discord.Message, tuple[discord.Message, ...]],
```

Comment:
`Union[discord.Message, tuple[discord.Message, ...]]` typehint is redundant here, a simple `tuple[discord.Message, ...]` will do. Even a single `discord.Message` passed in without the tuple will still be handled correctly, the command parser has a specialcase for this

Reply:
Alright, good to know